src/components/Settings/SettingsNotifications/NotificationModal/EmailModal.tsx

```
@@ -0,0 +1,504 @@
+  onClose,
+  onTest,
+  onSave,
+}: EmailModalProps) => {
+  const intl = useIntl();
+
+  const NotificationsEmailSchema = Yup.object().shape(
+    {
+      emailFrom: Yup.string()
+        .when('enabled', {
+          is: true,
+          then: Yup.string()
```

Biome (2.4.4)

[error] Do not add then to an object.

(lint/suspicious/noThenProperty)